src/OpenSage.Game/Diagnostics/AssetViews/ModelView.cs

```diff
@@ -34,7 +34,7 @@ void OnBuildingRenderList(object sender, BuildingRenderListEventArgs e)
                     true,
                     new MeshShaderResources.RenderItemConstantsPS
                     {
-                        HouseColor = context.Game.CivilianPlayer.Color.ToVector3(),
+                        HouseColor = Vector3.One, // context.Game.CivilianPlayer.Color.ToVector3(),
```

Comment:
I think I'll remove the commented code. I removed the Game-level `CivilianPlayer` because it seemed more correct to create that player along with all the others, when starting a game.